src/main/java/uk/gov/companieshouse/confirmationstatementapi/utils/Constants.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.companieshouse.confirmationstatementapi.utils;
+
+public class Constants {
+
+    private Constants() {}
+
+    public static final String ERIC_REQUEST_ID_KEY = "X-Request-Id";
```

Comment:
the eric request id key doesn't match the one being used in the logging interceptor, see if you can replace the one in logging interceptor with this one and make sure it still works.

Reply:
Checked and it's  not case-sensitive. So using it across.